DockStat.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
```

Comment:
if you're looking to gold-plate this: https://kvz.io/bash-best-practices.html

Reply:
this is a great resource - thank you! I suspect this will be a "one and done" kinda project, but if I go back an do a refactor or for my next project, I'll keep https://bash3boilerplate.sh/ in mind!